docs/time/index.rst

```diff
@@ -928,6 +932,24 @@ the TDB timescale::
 
     >>> time_barycentre = times.tdb + ltt_bary
 
+By default, the light travel time is calculated using the position and velocity
+of Earth and the Sun from built-in `ERFA <https://github.com/liberfa/erfa>`_ routines,
+but one can also use more precise calculations using the JPL ephemerides (which are derived from
+dynamical models). An example using the JPL ephemerides is:
+
+.. doctest-requires:: jplephem
+
+    >>> ltt_bary_jpl = times.light_travel_time(ip_peg, ephemeris='jpl') # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+    >>> ltt_bary_jpl # doctest: +REMOTE_DATA +FLOAT_CMP
+    <TimeDelta object: scale='tdb' format='jd' value=[-0.0037715  -0.00377286]> # doctest: +REMOTE_DATA +FLOAT_CMP
```

Comment:
You don't need to put the `# doctest` on the output lines (also the line below).